public/data/Noibisjunior.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "Abdulsalaam Noibi",
+  "type": "personal",
+  "bio": "FrontEnd Developer | OpenSource Contributor",
+  "avatar": "https://github.com/Noibisjunior.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Noibisjunior",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ClericCoder",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started learning to code",
+      "date": "February 2020",
+      "icon": "code",
+      "color": "green",
+      "description": "Started my software development bootcamp with freeCodeCamp",
+      "url": "https://github.com/Noibisjunior"
+    },
+    {
+      "title": "Started FreeLancing as a FrontEnd Developer",
+      "date": "September 2022",
+      "icon": "code",
+      "color": "blue",
+      "url": "Noibisjunior22@gmail.com"
```

It should be `mailto:Noibisjunior22@gmail.com`. Please update it with the correct syntax for the mail url! 
```suggestion
      "url": "mailto:Noibisjunior22@gmail.com"
```

---

You could also commit these changes directly by using the `Commit suggestion` button above!

---

great idea 👍